### ISSUE TITLE
Documentation: example output

### ISSUE THREAD

**Issue by mkhorton** [materialsproject/emmet#11]
Rather than trying to manually document them (which is proving time-consuming), I propose including 1 sample output document as JSON for each store used by the builders. I think it'll be a lot easier for debugging and for writing new builders.

**Comment by mkhorton** [comment]
These could also then be used as test files ...

**Comment by shyamd** [comment]
Yes, test files are badly needed. On my list once I get this small epoch of writing done.
